jdi-light-html-tests/src/test/java/io/github/epam/test/data/TableData.java

```diff
@@ -0,0 +1,13 @@
+package io.github.epam.test.data;
+
+import io.github.com.entities.UserInfo;
+
+public class TableData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TableData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5485505)